index.html

```diff
@@ -29,35 +29,36 @@
     <!--Main Page Content-->
     <header>
         <div class="logo">
-            <img class="mainlogo" src="/assets/pickFlix_logo3.png" alt="">
+            <img class="mainLogo" src="./assets/pickFlix_logo3.png" alt="PickFlix Logo">
         </div>
         <div class="landing">
             <div class="landingImage">
                 <img class="mainImage" src="assets/landingCover.png" alt="users enjoying a film">
             </div>
             <div class="landingText">
-                <img src="/assets/pickflix.png" alt="">  
+                <img src="./assets/pickflix.png" alt="PickFlix Logo">
```

Comment:
same with this. Add alt="PickFlix logo". I'm not sure if if the "." before the "/" makes a difference for you but mine doesn't work without the "."